chain-vote/src/committee.rs

```diff
@@ -54,6 +56,38 @@ impl ElectionPublicKey {
         );
         (ciphertexts, proof)
     }
+
+    /// Create an election public key from all the participants of this committee
+    pub fn from_participants(pks: &[MemberPublicKey]) -> Self {
```

Comment:
such a weird diff, this didn't change, right?

Reply:
No, I just grouped together all impl blocks for `ElectionPublicKey`, don't know why we had 2 in the same file